.github/workflows/nightly.yml

```diff
@@ -117,10 +117,95 @@ jobs:
         run: |
           cd "${BUILD_DIR}"
           cmake --build "${BUILD_DIR}" --target package
+      - name: Get version info
+        run: |      
+          cd ${SRC_DIR}
+          echo "VERSION=$(git describe --tags | sed 's/^release-//;s/-/+/;s/-/~/;s/rc/~rc/')-${MSYSTEM_CARCH}" >> $GITHUB_ENV
       - name: Package upload
         if: ${{ success() && matrix.btype == 'Release' && matrix.target == 'skiptest' }}
         uses: actions/upload-artifact@v3
         with:
-          name: darktable.${{ github.sha }}.win64.zip
+          name: darktable-${{ env.VERSION }}.zip
           path: ${{ env.BUILD_DIR }}/darktable-*.exe
```

Comment:
```suggestion
          name: darktable-${{ env.VERSION }}-${{ env.SYSTEM }}.exe
          path: ${{ env.BUILD_DIR }}/darktable-${{ env.VERSION }}-${{ env.SYSTEM }}.exe
```

Reply:
I hope this uploads the .exe installer directly, like the .dmg? There's really no need/benefit for zipping it...

Reply:
The github downloader first zips the file before the download starts so the dmg gets downoaded as .dmg.zip and the zip gets downloaded as .zip.zip
Unpacking the .zip.zip results in .exe

So replacing .zip by .exe is a really good idea...

Reply:
oops... the `path` is the path to the generated file, this must match. No need to change that.

Reply:
the `name` is what is displayed as the download file. Here you can set what you like.

Reply:
I wanted to change it, just to make sure the generated package filename is matching the expected upload one: https://github.com/darktable-org/darktable/actions/runs/4228028994/jobs/7343057807#step:8:36

Reply:
ok, that matches everything is fine

Reply:
which is in fact far better than `darktable-*.exe`